tools/qt-5.9.5-osx

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+# Reference:
+#   https://github.com/musescore/MuseScore/blob/master/build/travis/job_macos/install.sh
+
+echo Downloading Qt
+set -v
+set -e
+
+DOWNLOAD_URL=https://download.qt.io/archive/qt/5.9/5.9.5/qt-opensource-mac-x64-5.9.5.dmg
```

Comment:
It's a bummer we have to download this on every build, but pretty impressive that the extraction and environment build all works. Wow!

Reply:
I enabled caching of the Qt install.  The cache has to be downloaded from offsite just like the qt installer, but caching is about 11x faster than installing from scratch (650s -> 60s).